core/backend/src/Relationship.ts

```diff
@@ -291,7 +291,7 @@ export interface ElementDrivesElementProps extends RelationshipProps {
  *           e31
  * ```
  * # Callbacks
- * Once iModel.js has found the affected subgraph to process, it propagates changes through it by making callbacks.
+ * Once the affected subgraph to process are found, it propagates changes through it by making callbacks.
```

Comment:
```suggestion
 * Once the affected subgraphs to process are found, it propagates changes through it by making callbacks.
```

Reply:
```suggestion
 * Once the affected subgraph to process is found, it propagates changes through it by making callbacks.
```

Reply:
I'm going to change this to `Once the affected subgraph to process is found,` to make it all singular instead of plural.

Reply:
Let me know what you think of the change